src/api.ts

```diff
@@ -368,6 +352,7 @@ export function fernsRouter<T>(
           status: 400,
           title: error.message,
           error,
+          disableExternalErrorTracking: (error as any).disableExternalErrorTracking,
```

Comment:
A potential way to avoid this cast would be to make a helper function. I did a little playing around, and this is what I came up with:

```
/**
 * Safely extracts the disableExternalErrorTracking property from an error.
 * Works with both APIError instances and regular Error objects that may have
 * this property attached.
 */
export function getDisableExternalErrorTracking(
	error: unknown,
): boolean | undefined {
	if (error instanceof Error) {
		if (isAPIError(error)) {
			return error.disableExternalErrorTracking;
		}
	}
	if (
		error &&
		typeof error === "object" &&
		"disableExternalErrorTracking" in error
	) {
		return (error as { disableExternalErrorTracking?: boolean })
			.disableExternalErrorTracking;
	}
	return undefined;
}
```